contracts/test/MockSpokePool.sol

```diff
@@ -145,247 +145,6 @@ contract MockSpokePool is SpokePool, MockV2SpokePoolInterface, OwnableUpgradeabl
         chainId_ = _chainId;
     }
 
-    function depositV2(
-        address recipient,
-        address originToken,
-        uint256 amount,
-        uint256 destinationChainId,
-        int64 relayerFeePct,
-        uint32 quoteTimestamp,
-        bytes memory message,
-        uint256 // maxCount
-    ) public payable virtual nonReentrant unpausedDeposits {
-        // Increment count of deposits so that deposit ID for this spoke pool is unique.
-        uint32 newDepositId = numberOfDeposits++;
-
-        if (originToken == address(wrappedNativeToken) && msg.value > 0) {
-            require(msg.value == amount);
-            wrappedNativeToken.deposit{ value: msg.value }();
-        } else IERC20Upgradeable(originToken).safeTransferFrom(msg.sender, address(this), amount);
-
-        emit FundsDeposited(
-            amount,
-            chainId(),
-            destinationChainId,
-            relayerFeePct,
-            newDepositId,
-            quoteTimestamp,
-            originToken,
-            recipient,
-            msg.sender,
-            message
-        );
-    }
```

Comment:
We might want to retain this, tbf.

Reply:
Restored in a test capacity here: https://github.com/across-protocol/contracts/pull/663/commits/0186b0714467640f93f16882d5ebbf44f393d731